Lombiq.BaseTheme/Extensions/ResourceManagementOptionsExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Lombiq.BaseTheme.Constants;
+using OrchardCore.ResourceManagement;
+
+namespace Microsoft.Extensions.Options;
+
+public static class ResourceManagementOptionsExtensions
+{
+    private static readonly object _lock = new();
+
+    public static void RemoveBootstrap5(this IOptions<ResourceManagementOptions> resourceManagementOptions)
+    {
+        lock (_lock)
+        {
+            foreach (var manifest in resourceManagementOptions.Value.ResourceManifests)
+            {
+                if (manifest.GetResources("$" + nameof(FeatureIds.Area)).ContainsKey(FeatureIds.Area)) { continue; }
```

Comment:
Break this into multiple lines.

Reply:
I can just do this since we don't need to wrap braces in cs like in cshtml:
```suggestion
                if (manifest.GetResources("$" + nameof(FeatureIds.Area)).ContainsKey(FeatureIds.Area)) continue;
```

Reply:
OK!